Error connecting to database: Error: ER_ACCESS_DENIED_ERROR: Access denied for user 'root'@'localhost' (using password: NO)
    at Handshake.Sequence._packetToError (/home/wlgykctu/sensgro.rec22test.site/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Handshake.ErrorPacket (/home/wlgykctu/sensgro.rec22test.site/node_modules/mysql/lib/protocol/sequences/Handshake.js:123:18)
    at Protocol._parsePacket (/home/wlgykctu/sensgro.rec22test.site/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/home/wlgykctu/sensgro.rec22test.site/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/home/wlgykctu/sensgro.rec22test.site/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/home/wlgykctu/sensgro.rec22test.site/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/home/wlgykctu/sensgro.rec22test.site/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/home/wlgykctu/sensgro.rec22test.site/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (events.js:400:28)
    at addChunk (internal/streams/readable.js:293:12)
    --------------------
    at Protocol._enqueue (/home/wlgykctu/sensgro.rec22test.site/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/home/wlgykctu/sensgro.rec22test.site/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at Connection.connect (/home/wlgykctu/sensgro.rec22test.site/node_modules/mysql/lib/Connection.js:116:18)
    at Object.<anonymous> (/home/wlgykctu/sensgro.rec22test.site/sensor_data.js:15:12)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:101:18) {
  code: 'ER_ACCESS_DENIED_ERROR',
  errno: 1045,
  sqlMessage: "Access denied for user 'root'@'localhost' (using password: NO)",
  sqlState: '28000',
  fatal: true
}
Error inserting data into database: Error: Cannot enqueue Query after fatal error.
    at Protocol._validateEnqueue (/home/wlgykctu/sensgro.rec22test.site/node_modules/mysql/lib/protocol/Protocol.js:212:16)
    at Protocol._enqueue (/home/wlgykctu/sensgro.rec22test.site/node_modules/mysql/lib/protocol/Protocol.js:138:13)
    at Connection.query (/home/wlgykctu/sensgro.rec22test.site/node_modules/mysql/lib/Connection.js:198:25)
    at /home/wlgykctu/sensgro.rec22test.site/sensor_data.js:42:16
    at Layer.handle [as handle_request] (/home/wlgykctu/sensgro.rec22test.site/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/wlgykctu/sensgro.rec22test.site/node_modules/express/lib/router/route.js:144:13)
    at Route.dispatch (/home/wlgykctu/sensgro.rec22test.site/node_modules/express/lib/router/route.js:114:3)
    at Layer.handle [as handle_request] (/home/wlgykctu/sensgro.rec22test.site/node_modules/express/lib/router/layer.js:95:5)
    at /home/wlgykctu/sensgro.rec22test.site/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/home/wlgykctu/sensgro.rec22test.site/node_modules/express/lib/router/index.js:346:12) {
  code: 'PROTOCOL_ENQUEUE_AFTER_FATAL_ERROR',
  fatal: false
}
Error inserting data into database: Error: Cannot enqueue Query after fatal error.
    at Protocol._validateEnqueue (/home/wlgykctu/sensgro.rec22test.site/node_modules/mysql/lib/protocol/Protocol.js:212:16)
    at Protocol._enqueue (/home/wlgykctu/sensgro.rec22test.site/node_modules/mysql/lib/protocol/Protocol.js:138:13)
    at Connection.query (/home/wlgykctu/sensgro.rec22test.site/node_modules/mysql/lib/Connection.js:198:25)
    at /home/wlgykctu/sensgro.rec22test.site/sensor_data.js:42:16
    at Layer.handle [as handle_request] (/home/wlgykctu/sensgro.rec22test.site/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/wlgykctu/sensgro.rec22test.site/node_modules/express/lib/router/route.js:144:13)
    at Route.dispatch (/home/wlgykctu/sensgro.rec22test.site/node_modules/express/lib/router/route.js:114:3)
    at Layer.handle [as handle_request] (/home/wlgykctu/sensgro.rec22test.site/node_modules/express/lib/router/layer.js:95:5)
    at /home/wlgykctu/sensgro.rec22test.site/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/home/wlgykctu/sensgro.rec22test.site/node_modules/express/lib/router/index.js:346:12) {
  code: 'PROTOCOL_ENQUEUE_AFTER_FATAL_ERROR',
  fatal: false
}
Error inserting data into database: Error: Cannot enqueue Query after fatal error.
    at Protocol._validateEnqueue (/home/wlgykctu/sensgro.rec22test.site/node_modules/mysql/lib/protocol/Protocol.js:212:16)
    at Protocol._enqueue (/home/wlgykctu/sensgro.rec22test.site/node_modules/mysql/lib/protocol/Protocol.js:138:13)
    at Connection.query (/home/wlgykctu/sensgro.rec22test.site/node_modules/mysql/lib/Connection.js:198:25)
    at /home/wlgykctu/sensgro.rec22test.site/sensor_data.js:42:16
    at Layer.handle [as handle_request] (/home/wlgykctu/sensgro.rec22test.site/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/wlgykctu/sensgro.rec22test.site/node_modules/express/lib/router/route.js:144:13)
    at Route.dispatch (/home/wlgykctu/sensgro.rec22test.site/node_modules/express/lib/router/route.js:114:3)
    at Layer.handle [as handle_request] (/home/wlgykctu/sensgro.rec22test.site/node_modules/express/lib/router/layer.js:95:5)
    at /home/wlgykctu/sensgro.rec22test.site/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/home/wlgykctu/sensgro.rec22test.site/node_modules/express/lib/router/index.js:346:12) {
  code: 'PROTOCOL_ENQUEUE_AFTER_FATAL_ERROR',
  fatal: false
}
